src/sqlitestorage.cpp

```diff
@@ -198,63 +198,69 @@ bool SqliteStorage::open()
   sqlite3_busy_timeout( d->mDatabase, 1500 );
 
   /* Create Calendars, Components, etc. tables */
-  query = CREATE_VERSION;
-  sqlite3_exec( d->mDatabase );
+#ifdef Q_OS_UNIX
+  if (d->mSem.isInitialProcess()) {
```

Comment:
Do I need this?  How does the isInitialProcess() thing work?  Will the first process to attempt to acquire() the mutex be the initial process?

I'm just wondering if there's a possible problem here: if the process which hits this code, is NOT the initial process, but the initial process has not yet created the tables, then these CREATE queries will not be execd by this current process...  Is that possible, Matt?